pkg/api/universal/scheduler.go

```diff
@@ -83,7 +86,7 @@ func (a *Universal) DeleteJob(ctx context.Context, inReq *runtimev1pb.DeleteJobR
 		JobName: jobName,
 	}
 
-	_, err := a.schedulerClient.DeleteJob(ctx, internalDeleteJobReq)
+	_, err := a.schedulerManager.NextClient().DeleteJob(ctx, internalDeleteJobReq)
```

Comment:
I believe we want to rotate our scheduler clients in the case there is a req spike and not only if there is a connection err. Say we have 2 daprds, one schedules thousands of jobs a day, the other schedules 1 job once a month. In that case, we should cycle thru the schedulers so that we load balance our reqs better since the load from daprd is not equal.